addons/binding/org.openhab.binding.yeelight/README.md

```diff
@@ -38,6 +38,7 @@ All devices support some of the following channels:
 |`brightness` | `Dimmer` | This channel supports adjusting the brightness value, it is available on `dolphin` and `ceiling`.|
 |`color` | `Color` | This channel supports color control, it is available on `wonder` and `stripe`.|
 |`colorTemperature` | `Dimmer` | This channel supports adjusting the color temperature, it is available on `wonder` and `stripe` and `ceiling`.|
+|`command` | `String` | This channel sends command directly to device, it is available on all Yeelight Things.|
```

Comment:
"sends **a** command directly to **the** device"

Reply:
Fixed.